### ISSUE TITLE
General conversation thread

### ISSUE THREAD

**Issue by nrktkt** [kag0/json-anon#2]
continued from https://groups.google.com/g/json-rpc/c/l7m5SIvwnFs/m/nDM9pFc1AgAJ

**Comment by mpcm** [comment]
...  I only read the tail end of the thread and need to go back and read the actual thread before commenting more. But it strikes me that this is essentially the protobuf approach of numbering the names in named pairs, then assuming the other side has the schema/code details to reassemble them?

https://developers.google.com/protocol-buffers ... so the json would be [1234,"John Doe","jdoe@example.com"] ?

**Comment by nrktkt** [comment]
@mpcm yes exactly. but unlike protobuf the schema wouldn't have type information for each field.

the schema could be pre-shared, or it could be sent along with an array of anonymized objects.
